app/controllers/idv/choose_id_type_controller.rb

```diff
@@ -59,9 +58,9 @@ def chosen_id_type
 
     def set_passport_requested
       if chosen_id_type == 'passport'
-        idv_session.passport_requested = true
-      else
-        idv_session.passport_requested = false
+        document_capture_session.update!(passport_status: 'requested')
+      elsif document_capture_session.passport_requested?
```

Comment:
I don't follow this `elsif` logic - if the user didn't choose passport, but somehow the dcs already had passport_status == 'requested', then reset the status to 'allowed'? Why not just else? (passport is allowed per the before action)
```suggestion
      else
```